content/docs/rfcs/32/README.md

```diff
@@ -0,0 +1,614 @@
+---
+slug: 32
+title: 32/RLN
+name: Rate Limit Nullifier
+status: raw
+editor: Blagoj Dimovski <blagoj.dimovski@yandex.com>
+contributors: Barry Whitehat <barrywhitehat@protonmail.com>, Sanaz Taheri <sanaz@status.im>, Oskar Thorén <oskar@status.im>
```

Comment:
```suggestion
contributors: 
- Barry Whitehat <barrywhitehat@protonmail.com>
- Sanaz Taheri <sanaz@status.im>
- Oskar Thorén <oskar@status.im>
```
The CI errors, ad far as I can tell, are related to the format of contributors, they should be itemized. My suggested change should fix the error.